crates/matrix-sdk-crypto/src/store/memorystore.rs

```diff
@@ -136,6 +143,32 @@ impl MemoryStore {
     fn save_private_identity(&self, private_identity: Option<PrivateCrossSigningIdentity>) {
         *self.private_identity.write().unwrap() = private_identity;
     }
+
+    /// Return all the ['InboundGroupSession']s we have, paired with the
```

Comment:
Err, what?

```suggestion
    /// Return all the [`InboundGroupSession`]s we have, paired with the
```

Doesn't your text editor automatically highlight and linkify those if you use the correct syntax?

![image](https://github.com/matrix-org/matrix-rust-sdk/assets/552026/9a926cb2-416c-4604-b90c-440d5e6e4cff)